package.json

```diff
@@ -6,29 +6,41 @@
   "module": "dist/styled-media-query.es.js",
   "jsnext:main": "dist/styled-media-query.es.js",
   "repository": "git@github.com:morajabi/styled-media-query.git",
-  "keywords": ["styled-components", "media-query", "media", "breakpoint"],
+  "keywords": [
+    "styled-components",
+    "media-query",
+    "media",
+    "breakpoint"
+  ],
   "scripts": {
+    "test": "jest",
     "clean": "rimraf dist",
     "build": "yarn build:common && yarn build:es && yarn build:browser",
     "build:common": "rollup -c --environment COMMON,PRODUCTION",
     "build:es": "rollup -c --environment ES",
     "build:browser": "rollup -c --environment BROWSER,PRODUCTION"
   },
-  "files": ["dist", "src"],
+  "files": [
+    "dist",
+    "src"
+  ],
   "author": "Mohammad Rajabifard <mo.rajbi@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "styled-components": ">= 1 < 4"
+    "styled-components": ">= 4",
+    "react": ">= 16.3.0",
```

Comment:
`styled-media-query` doesn't depend on React and I think we should not require people to upgrade to avoid warnings when install this library.

Reply:
See my above comment: react & react-dom are meanwhile devDependency in styled-components. And it's needed for the tests anyhow...

Reply:
@timhagn, you mixed 2 PRs and it's not easy to review. I think this PR should not include any changes for tests which are done in #15.

Reply:
Will do, as soon as I'm up and running again (tomorrow / day after tomorrow).